book/src/implemented-proposals/durable-tx-nonces.md

```diff
@@ -118,6 +125,11 @@ one specified in the transaction's `recent_blockhash` field.
 If all three of the above checks succeed, the transaction is allowed to continue
 validation.
 
-### Open Questions
-
-* Should this feature be restricted in the number of uses per transaction?
+Since transactions that fail with an `InstructionError` are charged a fee and
+changes to their state rolled back, an opportunity for fee theft is presented
+in the case of durable nonce transactions as the `NonceAdvance` instruction's
+effects are reverted. This leaves a failed transaction free to be replayed and
```

Comment:
Suggest: "rolled back, there is an opportunity for fee theft if a `NonceAdvance` instruction is reverted. A malicious validator could replay the failed transaction and collect the fees repeatedly until the stored nonce is advanced."